app.js

```diff
@@ -1,11 +1,23 @@
 const express = require("express");
 const app = express();
 const endpointsJson = require("./endpoints.json");
+const getTopics = require("./controllers/topics.controllers");
 
 app.use(express.json());
 
-app.get("/api", (request, response) => {
-  response.status(200).send({ endpoints: endpointsJson });
+app.get("/api", (req, res) => {
+  res.status(200).send({ endpoints: endpointsJson });
```

Comment:
We want to make sure that we do still follow MVC pattern. We don't need a model for this one, but the controller should still handle the communication with the client :)